### DISCUSSION TITLE
gomcomplete extension ? (noob): compared to Jinja

### DISCUSSION THREAD

**Discussion by rirl** [hairyhenderson/gomplate#1381] [Q&A]
I am coming to gomplate from a background in Jinja and I am wondering if there is an extension API like that supported for Jinja, as described here https://jinja.palletsprojects.com/en/3.0.x/extensions/#jinja-extensions

**Comment by hairyhenderson** [comment]
Hi @rirl, thanks for trying out gomplate!

I'm not at all familiar with Jinja, so I'm not be the best person to compare it with gomplate, but it seems like [plugins](https://docs.gomplate.ca/config/#plugins) may be relevant. In concept, plugins are very simple - they're basically just a way to run an arbitrary command as if it were a regular gomplate function. 

What in particular are you looking to extend?
